cypress/e2e/marc/marc-authority/mark-auth-delete-auth-ref-record-with-linked-field-in-marc-bib.cy.js

```diff
@@ -0,0 +1,181 @@
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthoritiesDelete from '../../../support/fragments/marcAuthority/marcAuthoritiesDelete';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventoryViewSource from '../../../support/fragments/inventory/inventoryViewSource';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import { JOB_STATUS_NAMES } from '../../../support/constants';
+
+describe('MARC -> MARC Authority', () => {
+  const testData = {
+    tag100: '100',
+    tag150: '150',
+    tag650: '650',
+    searchOption: 'Keyword',
+    marcAuthIcon: 'Linked to MARC authority',
+    authorityType: 'Auth/Ref',
+    linked100FieldValues: [
+      11,
+      '100',
+      '1',
+      '\\',
+      '$a Chin, Staceyann, $d 1972-',
+      '$e Author $e Narrator',
+      '$0 id.loc.gov/authorities/names/n2008052404',
+      '$1 http://viaf.org/viaf/24074052',
+    ],
+    notLinked650FieldValues: [
+      19,
+      '650',
+      '\\',
+      '0',
+      '$a Feminist poetry $0 id.loc.gov/authorities/subjects/sh85047755',
+    ],
+    deleteModalMessage:
+      'Are you sure you want to permanently delete the authority record:  Poetry ? If you proceed with deletion, then 1 linked bibliographic record will retain authorized value and will become uncontrolled.',
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcBibFileForC374148.mrc',
+      fileName: `testMarcFileC374148.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      instanceTitle: 'Crossfire : a litany for survival.',
+      field650Value: 'The other side of paradise : a memoir / Staceyann Chin.',
+    },
+    {
+      marc: 'marcAuthFileForC374148_01.mrc',
+      fileName: `testMarcFileC374148.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      authority100FieldValue: '$a Chin, Staceyann, $d 1972-',
+      authorityTitle: 'Chin, Staceyann, 1972-',
+    },
+    {
+      marc: 'marcAuthFileForC374148_02.mrc',
+      fileName: `testMarcFileC374148.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      authorutyTitle: 'Poetry',
+      authority150FieldValue: 'Feminist poetry',
+    },
+  ];
+
+  const createdRecordIDs = [];
+
+  before('Create test data', () => {
```

Comment:
If test failed, next run will be failed too because some records won't be removed
Better remove test data in before section. See example C375224 (display-records-with-same-values-in-subject-field-and-linked-to-different-record.cy.js)